package.json

```diff
@@ -17,107 +17,98 @@
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.0.0",
     "@material-ui/core": "latest",
-    "autoprefixer": "^8.1.0",
     "au-table": "latest",
     "aurelia-animator-css": "^1.0.4",
     "aurelia-auth": "latest",
     "aurelia-bootstrapper": "^2.2.0",
     "aurelia-dependency-injection": "^1.2.0",
     "aurelia-fetch-client": "latest",
     "aurelia-history": "^1.1.0",
+    "aurelia-loader-nodejs": "latest",
     "aurelia-path": "^1.1.1",
     "aurelia-polyfills": "^1.3.0",
     "aurelia-router": "latest",
     "aurelia-templating": "latest",
     "aurelia-validation": "latest",
+    "aurelia-webpack-plugin": "latest",
+    "autoprefixer": "^8.1.0",
+    "babel-loader": "^8.0.0",
     "bluebird": "latest",
     "bootstrap": "latest",
+    "copy-webpack-plugin": "^4.5.1",
+    "css-loader": "^0.28.11",
     "csvjson": "latest",
     "dotenv": "latest",
+    "expose-loader": "^0.7.5",
+    "extract-text-webpack-plugin": "next",
+    "file-loader": "^1.1.11",
     "file-saver": "^1.3.8",
     "font-awesome": "4.7.0",
     "hammerjs": "^2.0.8",
+    "html-loader": "^0.5.5",
+    "html-webpack-plugin": "^3.0.6",
     "jquery": "^3.2.1",
     "material-ui-time-picker": "latest",
     "mini-css-extract-plugin": "^0.4.0",
+    "minimatch": "^3.0.4",
+    "nps": "latest",
+    "nps-utils": "latest",
     "popper.js": "^1.12.9",
     "postcss": "^6.0.14",
+    "postcss-loader": "latest",
     "postcss-sass": "^0.2.0",
     "react": "^16.3.0",
     "react-dom": "^16.3.0",
     "react-player": "^1.6.4",
     "react-redux": "^5.0.6",
     "redux": "^3.7.2",
-    "aurelia-loader-nodejs": "latest",
-    "aurelia-webpack-plugin": "latest",
-    "babel-core": "6.26.3",
-    "babel-loader": "^7.1.4",
-    "babel-plugin-syntax-flow": "^6.18.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1",
-    "babel-preset-stage-2": "^6.24.1",
-    "babel-register": "^6.26.0",
-    "copy-webpack-plugin": "^4.5.1",
-    "css-loader": "^0.28.11",
-    "expose-loader": "^0.7.5",
-    "extract-text-webpack-plugin": "next",
-    "file-loader": "^1.1.11",
-    "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^3.0.6",
-    "minimatch": "^3.0.4",
-    "nps": "latest",
-    "nps-utils": "latest",
+    "snyk": "latest",
     "style-loader": "latest",
     "through2": "^2.0.3",
     "ts-node": "^5.0.1",
     "uglify-js": "latest",
     "url-loader": "latest",
-    "postcss-loader": "latest",
     "wait-on": "latest",
     "webpack": "latest",
     "webpack-bundle-analyzer": "latest",
-    "webpack-cli": "latest",
-    "snyk": "latest"
+    "webpack-cli": "latest"
   },
   "peerDependencies": {},
   "devDependencies": {
+    "@babel/core": "^7.0.0",
```

Comment:
@DannyMcwaves this needs to be in dependancies, not devdependancies so our production build on Heroku does not fail